LibGUI/GWindow.cpp

```diff
@@ -307,6 +349,69 @@ void GWindow::event(CEvent& event)
     CObject::event(event);
 }
 
+void GWindow::paint_keybinds() {
+    if (m_keybind_mode) {
```

Comment:
Please make this an early return instead, to avoid nesting:
```suggestion
    if (!m_keybind_mode)
        return;
```